game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -191,14 +191,12 @@ private void gameDataChanged() {
   private Optional<ServerConnectionProps> getServerProps(final Component ui) {
     if (System.getProperty(TRIPLEA_SERVER, "false").equals("true")
```

Comment:
As a side note: The TRIPLEA_SERVER and TRIPLEA_CLIENT properties are some temporary properties as well.
AFAIK those are only being used to control the code flow in order to either create a server or a client i.e. host or join a game.
I know the properties are being used when trying to connect to or host a lobby game when we spin up a new instance of the JVM, but I'm not sure if they're used elsewhere.